README.md

```diff
@@ -1029,14 +1026,13 @@ Multiple constraints may be placed on a type variable by including multiple
 `TypeClass` values in the list (e.g. `{a: [Foo, Bar, Baz]}`).
 
 
-[1]: http://ramdajs.com/docs/#__
-[2]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments
-[3]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SyntaxError
-[4]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError
-[5]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MIN_SAFE_INTEGER
-[6]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER
-[7]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create
-[8]: https://en.wikipedia.org/wiki/Enumerated_type
-[9]: http://ramdajs.com/docs/#toString
-[10]: https://en.wikipedia.org/wiki/Semigroup
-[11]: https://github.com/fantasyland/fantasy-land#semigroup
+[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments
```

Comment:
I rebased this branch after merging #84. The changes to __README.md__ are much clearer now. Would you mind having another look?